### ISSUE TITLE
Fix for header in some Dialogs

### ISSUE THREAD

**Issue by jEsuSdA** [shimmerproject/Greybird#218]
I noticed something related with some dialogs (eg. Pulseaudio volume control applet for XFCE )

![captura de pantalla_2018-02-21_20-11-36](https://user-images.githubusercontent.com/5833446/36500591-02b3877c-1745-11e8-84a6-634856a99141.png)

As you can see, the dialog header background is gray, but almost all headers (like the Mime Editor) are white.

So, I tried to do a fix and this is what I get:

![captura de pantalla_2018-02-21_20-17-13](https://user-images.githubusercontent.com/5833446/36500683-36e7505a-1745-11e8-9c3f-45a5697d63cd.png)

I tried to find some ID or CLASS to apply, but I wont be able. I'm sure you can solve this issue in an elegant way. 

Thanks! ;)

Fix:

```
/* Fix for dialog header */
dialog > box widget
{
	background-color: #fff;
}

dialog > box > widget
{
	background-color: #fff;
	text-shadow:0 1px rgba(0, 0, 0, 0.15);
}
```

**Comment by jEsuSdA** [comment]
:( this patch causes interference with "filechooser". So, not recommended. :(

**Comment by ochosi** [comment]
I think your version of libxfce4ui is too old. This was fixed in version 4.13.1, more concretely in this patch:
https://git.xfce.org/xfce/libxfce4ui/commit/?id=67eb247599aa8380e01474a5677aaea63409015e

This should have nothing to do with Greybird.

**Comment by jEsuSdA** [comment]
I have the 4.12.1
Maybe this caused the problem.
Thanks! ;)
